tcms/testplans/static/testplans/js/get.js

```diff
@@ -103,3 +114,38 @@ function getTestCaseExpandArea(row, testCase) {
         }
     });
 }
+
+function attachEvents() {
+    // update default tester
+    $('.js-test-case-menu-tester').click(function(ev) {
+        // todo this prompt need to be translated
+        var email_or_username = window.prompt('Please type new email or username');
+        if (!email_or_username) {
+            return;
+        }
+        const testCaseId = getCaseIdFromEvent(ev);
+
+        // default_tester can be update only if we pass an user id
+        jsonRPC('TestCase.update', [testCaseId, {'default_tester': email_or_username}], function(res) {
+            console.log(res);
+        });
+    });
+
+    // delete testcase from the plan
+    $('.js-test-case-menu-delete').click(function(ev) {
+        const testCaseId = getCaseIdFromEvent(ev);
+        jsonRPC('TestPlan.remove_case', [testPlanId, testCaseId], function(res) {
+           $("div").find(`[data-testcase-pk=${testCaseId}]`).remove();
```

Comment:
I think this will not play well with my idea of using a `#tc-123` anchors mentioned below but what if you try `display: none` instead ? 

FTR in all of the other places I checked we do use .remove()